.devcontainer/environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - pandas
   - scipy
   - h5py
+  - nox
```

Comment:
Will also need `pytest`, `ipywidgets`, `torch`. Are there any others?

Reply:
I checked and all 3 of those are already included in the base image. The version of PANGEO that we chose includes PyTorch (with GPU support).